plenum/server/node.py

```diff
@@ -1941,7 +1941,10 @@ def postRecvTxnFromCatchup(self, ledgerId: int, txn: Any):
             self.poolManager.onPoolMembershipChange(txn)
         if ledgerId == DOMAIN_LEDGER_ID:
             self.post_txn_from_catchup_added_to_domain_ledger(txn)
-        rh = self.get_req_handler(ledgerId)
+        typ = get_type(txn)
+        # Since a ledger can contain txns which can be processed by an arbitrary number of request handlers;
```

Comment:
It looks like a general problem then? Should we remove the possibility of get req_handler by ledger?

Reply:
Probably, do you think we should have an exploratory ticket for this?

Reply:
It looks like a change of a concept in plenum, where it was assumed that there is just one req handler per ledger. There are a couple of other places where `get_req_handler(ledgerId)` is used (for example, `commitAndSendReplies` in node.py, which may affect some plugins).
So, yes, we need to create a ticket for this, remove possibility to get req handler by ledger id and cover it by unit tests.